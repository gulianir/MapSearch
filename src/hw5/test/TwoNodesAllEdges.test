# A complete 2 node graph with edges from every node

# Create a graph
CreateGraph testGraph

# Add two nodes
AddNode testGraph Node1
AddNode testGraph Node2

# Add all four edges
AddEdge testGraph Node1 Node1 Edge1
AddEdge testGraph Node1 Node2 Edge2
AddEdge testGraph Node2 Node1 Edge3
AddEdge testGraph Node2 Node2 Edge4

# Print the nodes and edges
ListNodes testGraph
ListChildren testGraph Node1
ListChildren testGraph Node2