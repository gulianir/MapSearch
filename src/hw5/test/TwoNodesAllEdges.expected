# A complete 2 node graph with edges from every node

# Create a graph
created graph testGraph

# Add two nodes
added node Node1 to testGraph
added node Node2 to testGraph

# Add all four edges
added edge Edge1 from Node1 to Node1 in testGraph
added edge Edge2 from Node1 to Node2 in testGraph
added edge Edge3 from Node2 to Node1 in testGraph
added edge Edge4 from Node2 to Node2 in testGraph

# Print the nodes and edges
testGraph contains: Node1 Node2
the children of Node1 in testGraph are: Node1(Edge1) Node2(Edge2)
the children of Node2 in testGraph are: Node1(Edge3) Node2(Edge4)